src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -798,6 +798,22 @@ private void AddKey(TermInfo.Database db, int keyId, ConsoleKey key, bool shift,
                     KeyFormatToConsoleKey[keyFormat] = new ConsoleKeyInfo('\0', key, shift, alt, control);
             }
 
+            private void AddPrefixKey(TermInfo.Database db, string extendedNamePrefix, ConsoleKey key)
+            {
+                AddKey(db, extendedNamePrefix + "3", key, false, true, false);
```

Comment:
nit: Named bool arguments might make the convention a little clearer here, i.e. I could tell that 3 is the suffix for "alt+key", 4 is for shift+alt+key, etc. at a glance.